modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -8088,6 +8092,32 @@ public void postProcessFile(File file, String fileType) {
         LOGGER.debug("Post processing file {} ({})", file, fileType);
     }
 
+    protected void executePostProcessor(String[] commandArr) {
```

Comment:
There were two different variants of this code. The majority without showing the error output in case the execution was non-zero. I unified them here, taking error output into account.